extensions/amp-analytics/0.1/vendors.js

```diff
@@ -247,9 +247,6 @@ export const ANALYTICS_CONFIG = /** @type {!JsonObject} */ ({
   },
 
   'bg': {
-    'transport': {
-      'iframe': 'https://tpc.googlesyndication.com/b4a/b4a-runner.html',
-    },
   },
```

Comment:
why keep this stanza here at all?

Reply:
Yeah...that's a holdover from before I merged the new object into the old one (below). The thinking was that keeping this maintain's the vendor ("bg")'s membership in the object. But now that the merge was implemented, that is moot. Will remove.

Reply:
Fixed